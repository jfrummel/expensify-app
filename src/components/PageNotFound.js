import React from 'react';
import { Link } from 'react-router-dom';

const PageNotFound = () => (
    <div>
        <h1>Status 404: Oops.. page not found</h1>
        <Link to="/">Go Home</Link>
    </div>
);

export default PageNotFound;