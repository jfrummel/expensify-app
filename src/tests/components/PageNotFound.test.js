import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from '../../components/PageNotFound';

test("renders not found page", () => {
    const wrapper = shallow(<PageNotFound />);
    expect(wrapper).toMatchSnapshot();
});