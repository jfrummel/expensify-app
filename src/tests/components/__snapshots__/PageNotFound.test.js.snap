// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders not found page 1`] = `
<div>
  <h1>
    Status 404: Oops.. page not found
  </h1>
  <[object Object]
    to="/"
  >
    Go Home
  </[object Object]>
</div>
`;
